stock_vertical_lift/models/vertical_lift_command.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import logging
+import threading
+
+from odoo import _, api, exceptions, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class VerticalLiftCommand(models.Model):
+    _name = "vertical.lift.command"
+    _order = "shuttle_id, name desc"
+    _description = "commands sent to the shuttle"
+
+    def _default_name(self):
+        return self.env["ir.sequence"].next_by_code("vertical.lift.command")
+
+    name = fields.Char(
+        "Name", default=lambda s: s._default_name(), required=True, index=True
+    )
+    command = fields.Char(required=True)
+    answer = fields.Char()
+    error = fields.Char()
+    shuttle_id = fields.Many2one("vertical.lift.shuttle", required=True)
+
+    def record_answer(self, answer):
+        name = self._get_key(answer)
+        record = self.search([("name", "=", name)], limit=1)
+        if not record:
+            if not getattr(threading.currentThread(), "testing", False):
```

Comment:
do we really need this?

Reply:
Thanks @simahawk for your review.
> @hailangvn what about this? Also, are we sure we always get values? We might have an empty dict AFAIR. Can you check?
I will work on this.